code/game/mecha/mecha.dm

```diff
@@ -610,10 +610,18 @@
 /obj/mecha/meteorhit()
 	return ex_act(rand(1,3))//should do for now
 
-/obj/mecha/emp_act(severity)
+/obj/mecha/emp_act(severity) // Severity is generally best left at 1; damage = (50 x severity), cell drain = ((current charge / 2) x severity)
```

Comment:
It's been reworked so that severity is a multiplied value, rather than a divided value. 

It makes the system considerably more logical, given that if you assign a higher number to severity you'd expect it to be more severe, not less.